volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -587,6 +587,15 @@
             ret = True
         elif flags_str == "r-x" and self.vm_file.dereference().vol.offset == 0:
             ret = True
+        elif "x" in flags_str:
+            for i in range(self.vm_start,self.vm_end,constants.linux.PAGE_SHIFT):
+                try:
+                    if proclayer.is_dirty(i):
+                        vollog.warning(f"Found malicious (dirty+exec) page at {hex(i)} !")
+                        ret = True
+                        break
+                except (exceptions.PagedInvalidAddressException, exceptions.InvalidAddressException):
```

Comment:
Also, please use vollog to log out something when an exception occurs, unless it's *expected* to occur.  Either way, a comment would be useful to say why it can be ignored (and it'll also make this messages go away).  5;)